pytorch_lightning/loops/fit_loop.py

```diff
@@ -186,7 +173,7 @@ def done(self) -> bool:
         if self.trainer.should_stop:
             # early stopping
             met_min_epochs = self.epoch_progress.current.processed >= self.min_epochs if self.min_epochs else True
-            met_min_steps = self.global_step >= self.min_steps if self.min_steps else True
+            met_min_steps = self.epoch_loop.global_step >= self.min_steps if self.min_steps else True
```

Comment:
just a little skeptical about this behavior change.
If you go with this, should we mention this in docs?

Reply:
Done. `max_step` should link now to `global_step` where we mention multiple optimizers and tbptt